rpm/create-rpm.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-thinking-things.
```

Comment:
I saw some references to `iotagent-context-adapter` in this file... I guess you mean `context-adapter` right? :)